### PR TITLE
Update rust container

### PR DESCRIPTION
**Pull request by TruongQuangSB**
- The [PR-build#5](https://github.com/eclipse-set/build/pull/5) should be merge and the workflows _Build and push docker image_ im build-repo should be successful before this PR
- Change rust docker image source to github container

### CODE REVIEW

**Comment by PetersNils** [comment]
/license-check
This is just a test unrelated to this PR to check if #10 worked.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,13 +4,27 @@ name: Build browser
...
 jobs:
+  get-organisation-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set name to lowercase
+        run: |
+          echo "ORGANISATION=${GITHUB_REPOSITORY_OWNER,,}" >> ${GITHUB_ENV}
+    outputs:
+      organisation: ${{ env.ORGANISATION }}
+
```
Comment (PetersNils) [inline .github/workflows/ci.yml:20]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,13 +4,27 @@ name: Build browser
...
   browser-jni:
     runs-on: ubuntu-latest
-    container: sbx320/rust-x86_64-pc-windows-gnu
+    needs: 
+      - get-organisation-name
+    container: ghcr.io/${{ needs.get-organisation-name.outputs.organisation }}/rust-x86_64-pc-windows-gnu:latest
```
Comment (PetersNils) [inline .github/workflows/ci.yml:25]:
```suggestion
    container: ghcr.io/eclipse-set/rust-x86_64-pc-windows-gnu:latest
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -4,13 +4,27 @@ name: Build browser
 
 on: push
 
+env:
+  REGISTRY: ghcr.io
+  ORGANISATION: ${{ github.repository_owner }}
+
```
Comment (PetersNils) [inline .github/workflows/ci.yml:10]: I guess we should get rid of these variables for simplicity.
```suggestion
```

**Review by PetersNils — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -4,13 +4,27 @@ name: Build browser
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set name to lowercase
+        run: |
+          echo "ORGANISATION=${GITHUB_REPOSITORY_OWNER,,}" >> ${GITHUB_ENV}
+    outputs:
+      organisation: ${{ env.ORGANISATION }}
+
   browser-jni:
     runs-on: ubuntu-latest
```
Comment (PetersNils) [inline .github/workflows/ci.yml:22]:
```suggestion
    runs-on: ubuntu-22.04
```

File: .github/workflows/ci.yml (hunk 3 above)
Comment (TruongQuangSB) [reply .github/workflows/ci.yml:10]: ok

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -4,13 +4,27 @@ name: Build browser
...
-    container: sbx320/rust-x86_64-pc-windows-gnu
+    needs: 
+      - get-organisation-name
+    container: ghcr.io/${{ needs.get-organisation-name.outputs.organisation }}/rust-x86_64-pc-windows-gnu:latest
     steps:
     - uses: actions/checkout@v3
-    
     - name: Set up Rust
       run: rustup target add x86_64-pc-windows-gnu
```
Comment (PetersNils) [inline .github/workflows/ci.yml:29]: This is already done within the docker container
```suggestion
```

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -6,13 +6,11 @@ on: push
...
 jobs:
   browser-jni:
-    runs-on: ubuntu-latest
-    container: sbx320/rust-x86_64-pc-windows-gnu
+    runs-on: ubuntu-22.04
+    container: ghcr.io/eclipse-set/rust-x86_64-pc-windows-gnu:latest
     steps:
     - uses: actions/checkout@v3
-    
     - name: Set up Rust
```
Comment (PetersNils) [inline .github/workflows/ci.yml:13]:
```suggestion
```

**Review by PetersNils — APPROVED** [review]
OK

**Comment by PetersNils** [comment]
Currently the required docker image is private and not accessible. I'll wait with merging this until the permissions are fixed.

**Review by PetersNils — APPROVED** [review]
With the Rust docker image now being public we can merge this.

**Pull request merged**